networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,57 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs):
++
+[source,terminal]
+----
+$ oc delete sriovnetwork -n openshift-sriov-network-operator --all
+$ oc delete sriovnetworknodepolicy -n openshift-sriov-network-operator --all
+$ oc delete sriovibnetwork -n openshift-sriov-network-operator --all
+----
+
+. Follow the instructions on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Deleting operators from a cluster] to remove the SR-IOV Network Operator from your cluster.
+
+. The SR-IOV resource definitions (CRDs) remain in the cluster after the SR-IOV Network Operator is uninstalled. Delete the remaining SR-IOV CRDs:
++
+[source,terminal]
+----
+$ oc get crd -o name | grep sriov | xargs oc delete
```

Comment:
Unfortunately we can not use this command in order to remove sriov crds. 
This command also can remove these resources:  **customresourcedefinition.apiextensions.k8s.io/sriovfecclusterconfigs.sriovfec.intel.com
customresourcedefinition.apiextensions.k8s.io/sriovfecnodeconfigs.sriovfec.intel.com**
Both these resources are part of Intel operator. 

May be we just can list resources that should be removed:
- customresourcedefinition.apiextensions.k8s.io/sriovibnetworks.sriovnetwork.openshift.io
- customresourcedefinition.apiextensions.k8s.io/sriovnetworknodepolicies.sriovnetwork.openshift.io
- customresourcedefinition.apiextensions.k8s.io/sriovnetworknodestates.sriovnetwork.openshift.io
- customresourcedefinition.apiextensions.k8s.io/sriovnetworkpoolconfigs.sriovnetwork.openshift.io
- customresourcedefinition.apiextensions.k8s.io/sriovnetworks.sriovnetwork.openshift.io
- customresourcedefinition.apiextensions.k8s.io/sriovoperatorconfigs.sriovnetwork.openshift.io